### PR TITLE
Android dev newarch imp req

### PR DESCRIPTION
**Pull request by oykuyilmaz1**
Please review the changes made for follow requests

### CODE REVIEW

**Comment by oykuyilmaz1** [comment]
Change the IP address

File: Platon/app/src/main/java/com/cmpe451/platon/adapter/NotificationElementsAdapter.kt [hunk 1]
```diff
@@ -65,9 +65,9 @@ class NotificationElementsAdapter(private val data: ArrayList<Notification>, pri
     /**
      * Removes element at given position
      */
-    fun removeElement(position: Int){
+    override fun removeElement(position: Int){
         data.removeAt(position)
-        this.notifyItemRemoved(position)
+        this.notifyDataSetChanged()
```
Comment (darktheorys) [inline Platon/app/src/main/java/com/cmpe451/platon/adapter/NotificationElementsAdapter.kt:70]: This seems good, but there could be a more beautiful solution.

File: Platon/app/src/main/java/com/cmpe451/platon/util/Definitions.kt [hunk 2]
```diff
@@ -24,7 +24,7 @@ import java.util.*
 class Definitions {
 
     companion object {
-        val API_URL="http://3.127.150.96:5000/"
```
Comment (darktheorys) [inline Platon/app/src/main/java/com/cmpe451/platon/util/Definitions.kt:27]: It shouldnot be hardcoded.

**Review by darktheorys — APPROVED** [review]
I have inspected the request, and tried it on my local emulator. Find no explicit errors or wrongly done things. It can be merged.

**Pull request merged**